selfdrive/car/CARS_template.md

```diff
@@ -1,6 +1,9 @@
 {% set footnote_tag = '[<sup>{}</sup>](#footnotes)' -%}
 {% set star_icon = '[![star](assets/icon-star-{}.svg)](##)' -%}
 {% set video_icon = '<a href="{}" target="_blank"><img height="18px" src="assets/icon-youtube.svg"></img></a>' -%}
+{% set width_tag = '<img width=2000>$<br>&nbsp;' %}
+{% set harness_col_name = (Column|list)[(Column|length) - 2].value %}
```

Comment:
the indexing we already do isn't great, we should probably clean that up later. you can just use the name from Python so you can find both instances when searching the codebase